src/fat12_partition/setup.rs

```diff
@@ -0,0 +1,9 @@
+static FS_DUMP: &[u8; 102400] = include_bytes!("../../dumps/linux_partitioned.dump");
+pub fn init(storage: &mut [u8], block_size: u32, blocks: u32) {
+    assert!(storage.len() >= (blocks * block_size) as usize);
+    assert!(storage.len() >= FS_DUMP.len());
+    unsafe {
+        let dst = storage.as_mut_ptr();
+        core::ptr::copy(FS_DUMP.as_ptr(), dst, FS_DUMP.len());
+    }
```

Comment:
Here we could avoid `unsafe` with a bit of:

```suggestion
    storage[..FS_DUMP.len()].copy_from_slice(FS_DUMP);
```


(and the two in [`fat12_partition/mod.rs`](https://github.com/chrisprice/pico-usb-mass-storage/blob/b6b49925bf0e442cf893092678527822a84483b2/src/fat12_partition/mod.rs#L174-L195))